rest_framework-stubs/fields.pyi

```diff
@@ -1,11 +1,13 @@
 import datetime
 import decimal
 import uuid
+from collections import OrderedDict
```

Comment:
Should we explicitly specify that it is `OrderedDict` and not just `dict`?
I guess that they have the same API.

I would be against this change.

Reply:
Using `Dict[]` is fine. Pushed an update for this.

I suppose `OrderedDict` is on its way to deprecation anyway, now that CPython's regular dict is now order-preserving.